### DISCUSSION TITLE
Accessing Electron's BrowserWindow Options

### DISCUSSION THREAD

**Discussion by WilliamWelsh** [microsoft/vscode-discussions#81] [Extension Development QnA] [answered]
Hello. I have an extension that makes the background a glass/acrylic/blur theme. I have an issue where, on MacOS, if the window loses focus, the glass aesthetic suddenly disables. I believe this is because, in VSCode, `BrowserWindow.options.visualEffectState` is set to the default of `followWindow` and I need to set it to `active` for the glass theme to not disable when focus is lost (like the default Terminal on Mac). I'm not sure how to edit the options of `BrowserWindow` through an extension. Is it possible?

**Comment by isidorn** [comment, marked as answer]
This is not possible to do from an extension.
